src/e3sm_io.c

```diff
@@ -242,6 +242,9 @@ int main (int argc, char **argv) {
             case 'c':
                 cfg.chunksize = atoll (optarg);
                 break;
+            case 'p':
+                cfg.profiling = 1;
```

Comment:
forgot to set the default value of cfg.profiling?

Reply:
fixed